README.md

```diff
@@ -59,3 +59,10 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 ###TESTING
 
 Run all from the testing tab in VSCode or poetry run pytest test
+
+
+#### Provision a VM from an Ansible Control Node
+
+copy the files from folder ansible_files/ to your host node then run
```

Comment:
"host" actually means the managed node. I guess because it hosts the app?

The word you were looking for is "controller".